### PR TITLE
PAAS-2642: Use provisioning API to handle RO/FRO

### PR DESCRIPTION
**Pull request by Ronan-Gorain**
JIRA issue: https://jira.jahia.org/browse/PAAS-2642

Short description:

**Don't forget** to update the README and any related documentation if needed: https://confluence.jahia.com/x/bYATAg

### CODE REVIEW

File: mixins/jahia.yml [hunk 1]
```diff
@@ -463,28 +463,71 @@ actions:
...
-    - cmd[${this.group}]: |-
-          max_try=15
-          try=0
-          until (ssh abricot@localhost -p 8101 -i /tmp/abricot -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR full-read-only ${this.fullreadmode}); do
-            # Failure can lead to a partial-on read-only mode. Consequently, we force read only to OFF
-            if [ $try == $max_try ]; then
-              ssh abricot@localhost -p 8101 -i /tmp/abricot -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null -o LogLevel=ERROR full-read-only OFF
-              exit 1
+    - cmd[proc]: |-
+        wc -l /opt/tomcat/logs/catalina.out |awk '{print $1}'
```
Comment (laurentfufu) [inline mixins/jahia.yml:472]:
```suggestion
        wc -l /opt/tomcat/logs/catalina.out |  awk '{print $1}'
```

Comment (Ronan-Gorain) [reply mixins/jahia.yml]: Fixed

File: mixins/jahia.yml [hunk 2]
```diff
@@ -1996,41 +2031,33 @@ actions:
...
+                path: /var/log/glusterfs/data.log
+                source: glusterfs
+                service: glusterfs
+            EOF
+            log_dir=/var/log/glusterfs
+            [ -d $log_dir ] || mkdir $log_dir
+            setfacl -m u:dd-agent:rx /var/log/glusterfs
+            setfacl -m u:dd-agent:rx /var/log/glusterfs/*.log
+            sed -i '/killall.*glusterd/a \  touch \/var\/log\/glusterfs\/glusterd.log && /usr\/bin\/setfacl -m u:dd-agent:rx \/var\/log\/glusterfs\/*.log' /etc/logrotate.d/glusterfs
+            systemctl restart rsyslog crond datadog-agent
```
Comment (julien-gagnon) [inline mixins/jahia.yml:2060]: I'm not sure I understand about this change in this PR. Could it be a mistake ?

**Review by julien-gagnon — CHANGES_REQUESTED** [review]

File: mixins/jahia.yml (hunk 2 above)
Comment (laurentfufu) [reply mixins/jahia.yml:2060]: I guess it is yes

Comment (laurentfufu) [reply mixins/jahia.yml]: should be ok now

**Review by julien-gagnon — APPROVED** [review]

**Pull request merged**